docs/89_glossary.md

```diff
@@ -402,7 +504,7 @@ reported tabulation outcome.
 
 * **county number** as part of the ballot manifest
```

Comment:
The definitions for "ballot manifest" and "cast vote record" should speak in terms of "ballot cards", rather than "ballots", they now do.
A ballot may be made up of multiple cards. A "ballot card" is a single sheet of paper, which may have contests on both sides. Each ballot card is tabulated independently, and reflected in the CVR report as a separate row. The "ballot style" for all cards of a ballot is either always or often the same for all of them, from the standpoint of the voter, though the software will consider the style to be the actual set of contests on the ballot card.